### PR TITLE
[BEAM-362] Move some easy stuff into runners/core-java

### PR DESCRIPTION
**Pull request by kennknowles**
Be sure to do all of the following to help us incorporate your contribution
quickly and easily:

 - [x] Make sure the PR title is formatted like:
   `[BEAM-<Jira issue #>] Description of pull request`
 - [x] Make sure tests pass via `mvn clean verify`. (Even better, enable
       Travis-CI on your fork and ensure the whole test matrix passes).
 - [x] Replace `<Jira issue #>` in the title with the actual Jira issue
       number, if there is one.
 - [x] If this contribution is large, please file an Apache
       [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.txt).

---

This change moves a set of classes with no dependents, leaving them in the same Java packages for now.

### CODE REVIEW

**Comment by kennknowles** [comment]
R: @bjchambers

File: sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReifyTimestampsAndWindows.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2015 Google Inc.
```
Comment (bjchambers) [inline sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReifyTimestampsAndWindows.java]: This is the wrong license. have you run `mvn verify` and/or whichever one verifies licenses and checkstyle?

Comment (kennknowles) [inline sdks/java/core/src/main/java/org/apache/beam/sdk/util/ReifyTimestampsAndWindows.java]: Done. And yea, I thought that was in the presubmit.

**Comment by bjchambers** [comment]
LGTM. self merge once Travis is happy if you want

**Pull request merged**